scripts/setup/flush-memcached

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh -xe
 
 # Flush memcached
-echo 'flush_all' | nc localhost 11211
+echo 'flush_all\nquit' | nc localhost 11211
```

Comment:
Hmm this seems to not work for me on Linux; I'm a bit surprised that memcached would have a different protocol based on OpenBSD vs. Linux, but I could imagine it being a version cutoff thing...
```
tabbott@monastery:~/zulip$ echo 'flush_all\nquit' | nc localhost 11211
ERROR
tabbott@monastery:~/zulip$ echo 'flush_all' | nc localhost 11211
OK
tabbott@monastery:~/zulip$ echo 'flush_all\nquit' | nc localhost 11211
ERROR
```